style.css

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+  padding: 0;
+
```

Comment:
Do not add blank lines, spaces, or indentations without a reason.

Reply:
- [ ] Polite suggestions are better than orders, again! You could phrase it like this `Please make sure to not add black lines, spaces or indentation without a reason, as it will make your code cleaner`.